searchit/scrapers/qwant.py

```diff
@@ -62,12 +63,29 @@ def _check_exceptions(self, res: ScrapeResponse) -> None:
     async def scrape(self, req: ScrapeRequest) -> List[SearchResult]:
         geo = req.geo if req.geo else "en_GB"
         urls = self._paginate(req.term, "", geo, req.count)
+        headers = {
```

Comment:
This doesn't actually get applied to the request - as on line 83 - the headers are overriden by the call to self.user_agent() - which is probably what should be providing the headers.